internal/component/faro/receiver/internal/payload/payload.go

```diff
@@ -238,6 +238,14 @@ func (m Measurement) KeyVal() *KeyVal {
 	}
 	MergeKeyVal(kv, m.Trace.KeyVal())
 	MergeKeyValWithPrefix(kv, KeyValFromMap(m.Context), "context_")
+
+	valuesAsString := make(map[string]string, len(m.Values))
+	for key, value := range m.Values {
+		valuesAsString[key] = fmt.Sprintf("%v", value)
```

Comment:
I am struggling a bit with this as well and @rlankfo also asked about it.

Atm all attributes of a Faro signal are converted to strings. 
I mostly did it to align with the attribute type of all the other signals.

But now that we are talking about it, it makes no sense to have a measurement to be string. 
It can only contain numbers, so let's keep it as a number.

Let me change it and align the receiver as well. 
cc @rlankfo @mar4uk